plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/model/LibraryFileTest.java

```diff
@@ -187,4 +187,19 @@ public void testEquals() {
     assertNotEquals(libraryFile1, null);
     assertNotEquals(libraryFile1, libraryFile1.getMavenCoordinates().toStringCoordinates());
   }
+  
+  @Test
+  public void testUpdateVersion() {
```

Comment:
It looks like this will cause a network access?  Can that be avoided?

(Or can we label these tests that require network access.  Perhaps even something as simple as `@Category({ NetworkInterface.class })` might work for now?)

Reply:
Maybe a good time to think about a new policy about tests with network connections. I know there are quite a lot of such tests. Put all of them into `.integration.appengine` going forward? Mark with `@Category`? Any other reasonable policy?

Reply:
We could use `Assume`.

JUnit 5, which has [conditions](http://junit.org/junit5/docs/current/user-guide/#extensions-conditions), or named tags like `@Tag("network")`.  But it requires Java 8… at least for the tests.